pkgs/applications/graphics/synfigstudio/default.nix

```diff
@@ -1,28 +1,46 @@
-{ stdenv, fetchurl, boost, cairo, fontsConf, gettext, glibmm, gtk3, gtkmm3
+{ stdenv, fetchFromGitHub, boost, cairo, fontsConf, gettext, glibmm, gtk3, gtkmm3
 , libjack2, libsigcxx, libtool, libxmlxx, makeWrapper, mlt-qt5, pango, pkgconfig
-, imagemagick, intltool
+, imagemagick, intltool, autoconf, automake, autoreconfHook, which
 }:
 
 let
-  version = "1.0.1";
+  version = "1.0.2";
 
   ETL = stdenv.mkDerivation rec {
     name = "ETL-0.04.19";
 
-    src = fetchurl {
-       url = "http://download.tuxfamily.org/synfig/releases/${version}/${name}.tar.gz";
-       sha256 = "1zmqv2fa5zxprza3wbhk5mxjk7491jqshxxai92s7fdiza0nhs91";
+    src = fetchFromGitHub {
+       repo   = "synfig";
+       owner  = "synfig";
+       rev    = version;
+       sha256 = "09ldkvzczqvb1yvlibd62y56dkyprxlr0w3rk38rcs7jnrhj2cqc";
     };
+
+    postUnpack = "sourceRoot=\${sourceRoot}/ETL/";
+
+    preConfigure = "autoreconf --install --force";
+    
+    buildInputs = [ autoconf automake ];
```

Comment:
Please use autoreconfHook and remove the `preConfigure` and the other buildInputs here.